### PR TITLE
Index `fronts.collections` as a list, not a string

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Adds some square brackets around the list of collections sent to ELK

## Why?
So we can do more interesting things with this data, like group by terms

### CODE REVIEW

File: facia/app/utils/DotcomFrontsLogger.scala [hunk 1]
```diff
@@ -27,7 +27,7 @@ case class DotcomFrontsLogger() extends GuLogging {
       ),
       LogFieldString(
         "front.collections",
-        faciaPage.collections.map(_.collectionType).mkString(", "),
+        faciaPage.collections.map(_.collectionType).mkString("[", ", ", "]"),
```
Comment (AshCorr) [inline facia/app/utils/DotcomFrontsLogger.scala:30]:
```suggestion
        JSON.stringify(faciaPage.collections.map(_.collectionType)),
```

might work better?

File: facia/app/utils/DotcomFrontsLogger.scala [hunk 2]
```diff
@@ -27,7 +27,7 @@ case class DotcomFrontsLogger() extends GuLogging {
       ),
       LogFieldString(
         "front.collections",
-        faciaPage.collections.map(_.collectionType).mkString(", "),
+        JSON.stringify(faciaPage.collections.map(_.collectionType)),
```
Comment (jamesgorrie) [inline facia/app/utils/DotcomFrontsLogger.scala:30]: Might need an import (can't do it my end, laptop froze).

Comment (jamesgorrie) [reply facia/app/utils/DotcomFrontsLogger.scala:30]:
```suggestion
        Json.stringify(Json.toJson(faciaPage.collections.map(_.collectionType))),
```

Comment (oliverlloyd) [reply facia/app/utils/DotcomFrontsLogger.scala:30]: Thanks

**Comment by AshCorr** [comment]
Oops, did not mean to close this!

**Comment by mxdvl** [comment]
@AshCorr I think it’s because your new branch is identical to the main branch?

**Comment by oliverlloyd** [comment]
I don't mind. You can close it if there's a better solution 👍

**Pull request closed without merging**